sdk-docs/project-structure.md

```diff
@@ -0,0 +1,93 @@
+## Table of Contents
+1. [Overview](#overview)
+2. [Main Directories](#main-directories)
+3. [Configuration Files In Depth](#configuration-files)
+
+## Overview
+### Difference between apps and extensions
+It is important to distinguish between an app and an extension as it might be confusing in the beginning.
+An "app" serves as a container in the marketplace, bundling one or more "extensions" to create an installable unit. The app handles configuration and metadata such as appearance in the marketplace, installation process, and descriptive information.
+
+On the other hand, an "extension" is where the main programming efforts are directed, as it contains the essential application logic and user interface. It is, in fact, the core functionality of the app.
+
+In simpler terms, think of the app as the external "shell" that brings everything together, while the extension is the heart of the app, where the functional aspects reside. It's crucial to grasp this distinction early on, but rest assured that it will become clearer as you delve deeper into app development.
+
+[comment]: <Should put an illustration that depicts the above in a simple manner>
+
+## Main Directories
+[comment]: <Visual representation of the structure (tree diagram or screenshot)>
+
+### apps
+The `apps` directory houses the main application projects within the mono-repo. Each app project resides in its own subdirectory. The apps can share code and resources from the `libs` directory, streamlining the development process and reducing code duplication. The following files are important to understand and are created after generating an app:
+- `/[app_name]`
+    - `/src`
+        - `index.ts`: Leave it empty as code will be injected to this file automatically when you run your app. 
+    - `iris-app-manifest.ts`: This is a very important file that keeps all your configuration, such as marketplace configurations, activationMode, extensions, etc.
+    - `package.json`: node dependencies for only this app.
+    - `project.json`: holds project-specific settings and build configurations and is used by the Nx CLI to understand how to build, test, and serve your application or library.
+    Beyond what our guides tell you to do, you likely won't have to look too much into this file.
+    - `webpack.config.ts`: configuration file used by Webpack, a popular open-source JavaScript module bundler. Under normal circumstances you will never have to modify this file.
+
+### libs
+The libs folder should contain all the extensions, tools and helper-functions that you might want to share between apps. When you generate your first extension, you will have a folder structure looking roughly like this:
```

Comment:
This folder contains all `extensions`, which can be shared between `apps`. Developers can also create normal NX libraries (not extensions) for tools, helper functions, and other shared code such as React components to share between `extensions`. 😊